src/pycram/failure_handling.py

```diff
@@ -1,8 +1,11 @@
 from .designator import DesignatorDescription
 from .plan_failures import PlanFailure
+from threading import Lock
+from typing import Union
```

Comment:
Import from typing_extension instead, names stay the same, python version compatibility increases

Reply:
Oh i overlooked that one, should be fixed as soon as Vanessa merges my PR

Reply:
done